usaspending_api/search/delta_models/award_search.py

```diff
@@ -637,3 +633,39 @@
     faba.award_id
 ) TREASURY_ACCT ON (TREASURY_ACCT.award_id = awards.id)
 """
+
+award_search_incremental_load_sql_string = [
+    f"""
+    CREATE OR REPLACE TEMPORARY VIEW temp_award_search_view AS (
+        {_base_load_sql_string}
+    )
+    """,
+    f"""
+    MERGE INTO rpt.award_search AS t
+    USING (SELECT * FROM temp_award_search_view) AS s
+    ON t.award_id = s.award_id
+    WHEN MATCHED AND
+      ({" OR ".join([f"NOT (s.{col} <=> t.{col})" for col in AWARD_SEARCH_DELTA_COLUMNS])})
+      THEN UPDATE SET *
+    WHEN NOT MATCHED THEN INSERT *
+    -- The following line can replace the second delete after upgrading to Databricks runtime 12.1
+    -- https://docs.databricks.com/en/sql/language-manual/delta-merge-into.html
+    -- WHEN NOT MATCHED BY SOURCE THEN DELETE
+    """,
+    """
+    DELETE FROM rpt.award_search
+    WHERE award_id IN (
+      SELECT t.award_id
+      FROM rpt.award_search AS t
+      LEFT ANTI JOIN int.awards AS s ON t.award_id = s.id
+    );
+    """,
+]
+
+award_search_load_sql_string = f"""
```

Comment:
This variable name could be better. It's pretty ambiguous considering the other load query has "incremental" in the variable name.

Because I've taken the time to read the changes I know this is the "non-incremental" version. However, the current variable name does not indicate that. I suggest updating the variable to be more explicit.

Reply:
Good idea here. Updated to be `award_search_overwrite_load_sql_string`